code/lab1-tdd-arabic-roman-converter/src/test/java/ru/unn/agile/ArabicRomanConverter/ArabicToRomanTest.java

```diff
@@ -0,0 +1,157 @@
+package ru.unn.agile.ArabicRomanConverter;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ArabicToRomanTest {
+
+    private String convertToRoman(final int arabic) {
+        ArabicToRomanConverter converter = new ArabicToRomanConverter();
+        return converter.convert(arabic);
+    }
+
+    @Test
+    public void canConvertToEmptyString() {
+        String romanNumber = convertToRoman(0);
+        assertEquals("", romanNumber);
+    }
+
```

Comment:
The same in negative cases when you got -1 for instance, add test on negative int. 
You can use @Test(expected = IllegalArgumentException.class) construction in test to expect exception.

Reply:
Done